app/_src/mesh/features/provenance-verification-binaries.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Verify Build Provenance for Kong Mesh Binaries
+badge: enterprise
+---
+
+Starting with 2.7.4, {{site.mesh_product_name}} produces build provenance for binary artifacts, which can be verified using `cosign` / `slsa-verifier` with attestations published to a Docker Hub repository.
+
+This guide provides steps to verify build provenance for signed {{site.mesh_product_name}} binary artifacts in two different ways:
+
+* A minimal example, used to verify an binary artifacts without leveraging any annotations
+* A complete example, leveraging optional annotations for increased trust
+
+For the minimal example, you only need a compressed binary file and provenance file.
+
+For the complete example, you need the same details as the minimal example, as well as any of the optional annotations you wish to verify:
+
+| Shorthand | Description | Example Value |
+|---|---|---|
+| `<repo>` | GitHub repository | `kong-mesh` |
+| `<workflow name>` | GitHub workflow name | `build-test-distribute` |
+| `<workflow trigger>` | Github workflow trigger name | `push` |
+| `<version>` | Artifact version to download | `2.7.4` |
+| `<binary-files>` | Compressed binary files for the specified version | `kong-mesh-<version>-*-*.tar.gz` |
+| `<provenance-file>` | Binary provenance file | `kong-mesh.intoto.jsonl` |
+
+Because Kong uses GitHub Actions to build and release, Kong also uses GitHub's OIDC identity to generate build provenance for binary artifacts, which is why many of these details are GitHub-related.
+
+## Examples
+
+### Prerequisites
+
+For both examples, you need to:
+
+1. Ensure `slsa-verifier` is installed.
+
+2. [Download security assets](https://cloudsmith.io/~kong/repos/kong-mesh-binaries-preview/packages/?q=name%3Asecurity-assets*+version%3A%3E%3D2.7.4) for the required version of {{site.mesh_product_name}} binaries 
```

Comment:
Link to all SLSA assets with `name:security-assets version:>=2.7.4` to filter cloudsmith packages under `kong-mesh-binaries-release`.

`Preview Security Assets for Testing`: https://cloudsmith.io/~kong/repos/kong-mesh-binaries-preview/packages/?q=name%3Asecurity-assets*+version%3A%3E%3D2.7.4)